internal/blockchain/fabric/fabric.go

```diff
@@ -334,6 +336,10 @@ func (f *Fabric) handleBatchPinEvent(ctx context.Context, msgJSON fftypes.JSONOb
 	})
 }
 
+func (f *Fabric) buildEventLocationString(msgJSON fftypes.JSONObject) string {
+	return fmt.Sprintf("chaincode=%s", msgJSON.GetString("chaincodeId"))
```

Comment:
Have I understood correctly that we're using a `x=y` querystring like syntax for Fabric, and a serialized JSON object for Ethereum? Wondering why that choice

For Fabric, how is the `channel` represented?

Reply:
This particular string is used for Prometheus metrics filtering. It would ideally include the channel as well, but there's no easy way to get it in the context where this is called (and seems like not a major issue as it only affects metrics). Ethereum has a similar "address=0x0000" format for this same purpose.

The serialization of the location string for storage is JSON and is consistent across Ethereum/Fabric. But yes, we do have two different serializations of the same info for different purposes.

Reply:
Thanks for the clarification. Understood 👍